docs/releasing.rst

```diff
@@ -38,7 +38,7 @@ information so we can reference it later.
         -volname "RktMachine" \
         -fs HFS+ -fsargs "-c c=64,a=16,e=16" \
         -format UDRW \
-        -megabytes 96 \
+        -megabytes 128 \
```

Comment:
Nah, you're right. The git clone/push is pretty slow now with this image file checked in. I've added a commit to this review to compress the contents of the image file which will help.

The long term solution is to make tools.qcow2 as part of the build process and remove it from git. I've restructured the operations in `building.rst` to make this a little more likely, e.g. less switching between the container and the host.

In the meantime I'll purge the unneeded tools.qcow2 objects from the git repository history.

```
$ du -sh src/vm/tools.qcow2
 74M	src/vm/tools.qcow2
$ du -sh src/vm/tools.qcow2
 23M	src/vm/tools.qcow2
```